test/deprecated-keyword/handleConfigParam.chpl

```diff
@@ -0,0 +1,7 @@
+// Handle when a config param is referenced via its flag
+deprecated "don't use x, use y" config param x: bool = false;
```

Comment:
`deprecated` can also not take a string, right?  How confident are you about the orthogonality of the code such that, if no string is used, the compile-time and execution-time warnings will also print as expected?

Reply:
Very confident, but it's also not a burden to add a test in this case.

Reply:
I don't feel strongly about it if you feel confident that it's orthogonal w.r.t. other existing tests.